### DISCUSSION TITLE
Community Office Hours

### DISCUSSION THREAD

**Discussion by fniedtner** [microsoft/autogen#4059] [Announcements]
We are starting a new series of community office hours, a weekly invitation for everyone to join and ask questions, discuss ideas and share insights.

Join us on Wednesdays 8am PST/11am EST/5pm CET at https://teams.microsoft.com/meet/2344899661716?p=bIxbaT1oqHeFsbPqT8

**Comment by diegolescano** [comment]
Hi team, regarding documentation, I see some items such as Managing State in https://microsoft.github.io/autogen/dev/user-guide/agentchat-user-guide/quickstart.html, but I don't see a dev branch to pull and validate the code against the documentation. The latest version to pull is 0.4.0.dev8 and it doesn't contain Managing State. How should I proceed to validate that the documentation in the like provided can be executed by a final user? Thanks!

**Reply by gagb** [reply]
Here is the code Diego, it's in the main branch. So should be there in the next dev release. cc @ekzhu @victordibia  https://microsoft.github.io/autogen/dev/reference/python/autogen_agentchat.state.html

**Reply by ekzhu** [reply]
@diegolescano you can switch the documentation version using the top-left drop-down menu. 

![image](https://github.com/user-attachments/assets/646eecb2-763b-419c-ade9-e9aff11d1df8)

**Reply by hamdy4949** [reply]
عرفني بنفسك ماذا تستطيع أن تفعل

**Comment by gatorwatt** [comment]
Hi there, I sat your NeurIPS tutorial online yesterday and was playing around a little with the video surfer notebook. Had a little difficulty logging into the discord link so thought that I might try to say hello at one of your upcoming office hours. Currently looking for new opportunities and just wanted to introduce myself. Regards, Nicholas Teague

**Comment by fniedtner** [comment]
> We are starting a new series of community office hours, a weekly invitation for everyone to join and ask questions, discuss ideas and share insights.
> 
> Join us on Wednesdays 10am PST/1pm EST/7pm CET at https://teams.microsoft.com/meet/222836470934?p=3bs3pOaAtlftpTiCRh.

UPDATE: As of next week the office hour will be held on discord: https://discord.gg/NWM9SE4v?event=1316835803007156345

**Reply by vfthome-leasea** [reply]
Can you send me a new link for the weekly sessions?

**Comment by Tulingli** [comment]
Does Autogen support non-OpenAI models and how should they be used?

**Reply by ekzhu** [reply]
For this please create a separate discussion thread.

You can read our documentation, which describes local models and other openai-compatible models.

**Comment by VenkateshSrini** [comment]
Hi All,
I'm from India and really interested in running AutoGen .NET version of Agents in Kubernetes with Grpc. I have posted by ask in the discussion. Since we are in India, we are unable to connect to for office hours. Is there way we can connect during  early US hours to discuss our ask

**Reply by rysweet** [reply]
Hi @VenkateshSrini would be happy to discuss with you. I am often available from 5am PT onward - if you are MSFT send me a teams message - if you aren't MSFT, then start a discussion thread with your questions here and I'll do what I can to help and maybe we can do a one-off office hours that is in an India friendly time slot.

**Reply by VenkateshSrini** [reply]
> Hi @VenkateshSrini would be happy to discuss with you. I am often available from 5am PT onward - if you are MSFT send me a teams message - if you aren't MSFT, then start a discussion thread with your questions here and I'll do what I can to help and maybe we can do a one-off office hours that is in an India friendly time slot.

I have started a thread here https://github.com/microsoft/autogen/discussions/5766

**Reply by VenkateshSrini** [reply]
Hi @rysweet did you get a chance to see the above discussion thread ?

**Comment by vacha-y-dave-usps** [comment]
Hello, I am looking to integrate Microsoft AD with autogen studio, Can someone from autogen team please provide guidance on how can one do this? Currently Autogen Studio has a "guest" user account, I would like to create login screen and enable authentication through AD user groups.

**Reply by victordibia** [reply]
@vacha-y-dave-usps , we have an experimental login/auth mechanism that supports GH auth.
https://microsoft.github.io/autogen/dev/user-guide/autogenstudio-user-guide/experimental.html#authentication 

The way to integrate AD would be to [implement](https://github.com/microsoft/autogen/pull/5928) a similar underlying AD based system by extending the auth config infra.

**Reply by vacha-y-dave-usps** [reply]
Thanks so much @victordibia , I am going to try Git based auth as a starting point. Appreciate your quick response.

**Comment by vfthome-eleme** [comment]
Hi guys, do we still have the weekly meetings? Or any meetings at all that I can chat or watch live content about autogen?

We are considering the adoption of Autogen for a big company project that we are creating, and this exchange would be valuable to consider the general feedbacks and active community before using it on a bigger project..

**Reply by ekzhu** [reply]
Yes we do. The link has recently been updated. We just had one today.

**Reply by vfthome-eleme** [reply]
@ekzhu can you send me an inv if you guys have an disc server too, would be great to participate as a listener or helper.

**Reply by ekzhu** [reply]
See repo homepage for the discord link thanks!

**Reply by vfthome-eleme** [reply]
Thanks, got it!

**Comment by rotodd** [comment]
Hey, just checking in - are the weekly office hours still ongoing? Keen to attend the next one
